spec/bolt/transport/winrm_spec.rb

```diff
@@ -14,8 +14,10 @@
   include BoltSpec::Task
 
   def mk_config(conf)
-    stringified = conf.each_with_object({}) { |(k, v), coll| coll[k.to_s] = v }
-    Bolt::Config.new(transport: 'winrm', transports: { winrm: stringified })
+    unstringified = conf.each_with_object({}) { |(k, v), coll| coll[k.to_s] = v }
```

Comment:
Why is this *un*stringified now?

Reply:
Oh, neither name actually makes much sense. `stringified` was confusing. I can try to come up with a new one.

Reply:
Oh right, I forgot what i was doing. Turning keys into strings.